src/app/clusters/operational-credentials-server/operational-credentials-server.cpp

```diff
@@ -446,7 +447,7 @@ bool emberAfOperationalCredentialsClusterRemoveFabricCallback(app::CommandHandle
     else
     {
         ChipLogProgress(Zcl, "OpCreds: RemoveFabric successful");
-        SendNOCResponse(commandObj, commandPath, OperationalCertStatus::kSuccess, fabricBeingRemoved, CharSpan());
+        SendNOCResponse(commandObj, commandPath, NodeOperationalCertStatusEnum::kOk, fabricBeingRemoved, CharSpan());
```

Comment:
Not sure how strict we are with this, but spec has it as "OK", so should this be **kOK**?

Reply:
Spec has "OK", and XML has "OK", but the cluster-objects codegen does not preserve acronym casing.  We can try to change that, but it would have to be a separate PR...

Reply:
Ok, I have no strong opinion on this. I just noticed that there were some places where it used OK and others where is used Ok, but if kOk is auto generated then it is what it is.

Reply:
I filed https://github.com/project-chip/connectedhomeip/issues/24485 on this.